spec/spec_helper_acceptance_local.rb

```diff
@@ -58,22 +79,97 @@ def return_host
   master.uri
 end
 
-def standard_manifest
+def default_splunk_class
   <<-MANIFEST
-    service { 'pe-puppetserver':
-      ensure     => running,
-      hasrestart => true,
-      restart    => "service pe-puppetserver reload",
-    }
-    class { 'splunk_hec':
+    class{'splunk_hec':
       url                    => "http://#{master.uri}:8088/services/collector/event",
       pe_console             => "#{master.uri}",
-      splunk_token           => "abcd1234",
-      pe_username            => "admin",
+      splunk_token           => 'abcd1234',
+      pe_username            => 'admin',
       pe_password            => Sensitive('pie'),
       enable_reports         => true,
       manage_routes          => true,
       include_api_collection => true,
     }
   MANIFEST
 end
+
+def token_splunk_class
+  <<-MANIFEST
+    class{'splunk_hec':
+      url                    => "http://#{master.uri}:8088/services/collector/event",
+      pe_console             => "#{master.uri}",
+      splunk_token           => 'abcd1234',
+      pe_username            => 'admin',
```

Comment:
is pe_username required here?

Reply:
Nope! And removing it helps make sure we catch edge cases related to that too.